en/ros/ros2_comm.md

```diff
@@ -178,12 +178,27 @@ The PX4 simulator starts the uXRCE-DDS client automatically, connecting to UDP p
 To start the simulator (and client):
 
 1. Open a new terminal in the root of the **PX4 Autopilot** repo that was installed above.
-1. Start a PX4 [Gazebo](../sim_gazebo_gz/README.md) simulation using:
 
-   ```sh
-   make px4_sitl gz_x500
-   ```
-   
+   :::: tabs
+
+   ::: tab humble
+   - Start a PX4 [Gazebo](../sim_gazebo_gz/README.md) simulation using:
```

Comment:
FYI, we use names Gazebo and Gazebo Classic everywhere to match the Gazebo team.